install_rundeps.sh

```diff
@@ -35,7 +35,7 @@ install_run_utils() {
         dnsutils \
 		iproute2 \
 		iputils-ping \
-		tcpdump
+		tcpdump vim
```

Comment:
not sure if vim should be part of build. Its ok to keep it during your development but not in the actual production build.

Reply:
OK.. removed